configs/defs/generic_linux.textproto

```diff
@@ -20488,3 +20773,96 @@ benchmark_configs: {
       "}}"
   }
 }
+
+benchmark_configs: {
+  id: "accounts-in-etc-passwd-use-shadowed-passwords"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/o:debian:debian_linux:12" version: "1.1.0" benchmark_document: "CIS Debian Linux 12" }
+    version: { cpe_uri: "cpe:/o:centos:centos:7.%02" version: "4.0.0" benchmark_document: "CIS Centos Linux 7.x" }
```

Comment:
This isn't used by CentOS - We should probably remove the CentOS version and move this to conigs/defs/debian.textproto

Reply:
Sounds good, let's add the Rocky version then